provider/openstack/provider.go

```diff
@@ -1779,46 +1779,36 @@ func (e *Environ) Instances(ctx context.ProviderCallContext, ids []instance.Id)
 	if len(ids) == 0 {
 		return nil, nil
 	}
-	// Make a series of requests to cope with eventual consistency.
-	// Each request will attempt to add more instances to the requested
-	// set.
-	var foundServers []nova.ServerDetail
-	for a := shortAttempt.Start(); a.Next(); {
-		var err error
-		foundServers, err = e.listServers(ctx, ids)
-		if err != nil {
-			logger.Debugf("error listing servers: %v", err)
-			if !IsNotFoundError(err) {
-				handleCredentialError(err, ctx)
-				return nil, err
-			}
-		}
-		if len(foundServers) == len(ids) {
-			break
+
+	foundServers, err := e.listServers(ctx, ids)
+	if err != nil {
+		logger.Debugf("error listing servers: %v", err)
+		if !IsNotFoundError(err) {
+			handleCredentialError(err, ctx)
+			return nil, err
 		}
```

Comment:
Should we be debugging the error if it is NotFound since we actually handle that error (we don't trigger failures, etc)